README.adoc

```diff
@@ -174,15 +184,17 @@ The helper function invokes the `system` service by calling the `defaultRestClie
 // =================================================================================================
 == Building the client with RestClientBuilder
 
-The `inventory` service might also provide a different host name rather than the default `localhost` host name. Set the host name as a variable and build the client by using the `RestClientBuilder` method.
+The `inventory` service might also want to connect with a host other than the default `localhost`. Then, it is not reasonable to configure a base URL that is not yet known.
```

Comment:
The word "want" sounds too anthropomorphic to me. What about just "service might also connect"? Or maybe "service can connect"?

Also add "host name" (or another label word) after `localhost`.

"The `inventory` service **can also connect** with a host other than the default `localhost` **host name**."

I'd rewrite the second sentence as,
"Do not configure a base URL that is not yet known."